OpenRA.Mods.RA/Traits/PortableChrono.cs

```diff
@@ -108,19 +105,12 @@ public bool CanTeleport
 			get { return chargeTick <= 0; }
 		}
 
-		public IEnumerable<PipType> GetPips(Actor self)
+		public float GetValue()
 		{
-			for (var i = 0; i < Info.Pips; i++)
-			{
-				if ((1 - chargeTick * 1.0f / (25 * Info.ChargeTime)) * Info.Pips < i + 1)
-				{
-					yield return PipType.Transparent;
-					continue;
-				}
-
-				yield return PipType.Blue;
-			}
+			return (float)((chargeTick - (25 * Info.ChargeTime)) * (Info.ChargeTime * -0.0001));
```

Comment:
Scaling the charge time based on game speed doesn't make any sense (this breaks the game balance at different speeds).  The proper fix is to convert the charge time to ticks so that it is unambiguously measured relative to the world.